sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/DistanceFunction.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+/**
+ * Distance Function for the embeddings in the Cosmos DB database service.
+ */
+public enum DistanceFunction {
+    /**
+     * Represents the euclidean distance function.
+     */
+    EUCLIDEAN("euclidean"),
+
+    /**
+     * Represents the cosine distance function.
+     */
+    COSINE("cosine"),
+
+    /**
+     * Represents the dot product distance function.
+     */
+    DOT_PRODUCT("dotproduct");
+
+    private final String overWireValue;
+
+    DistanceFunction(String overWireValue) {
+        this.overWireValue = overWireValue;
+    }
+
+    @Override
+    public String toString() {
+        return this.overWireValue;
+    }
+
+    /**
+     *
+     * @return value for the enum
+     */
+    public String getValue() {
```

Comment:
we don't need this API as toString() is enough for this enum as it returns the same overWireValue
The less the public surface area, the better to start with.